### PR TITLE
Change frontend post requests parameters to be sent in body

### PR DESCRIPTION
**Pull request by ahmad-shawky**
This pull request change frontend post requests parameters to be sent in body. currently post requests params sent with url which causing some firewals block the requests because CONTENT-LENGTH header sent with zero

### CODE REVIEW

**Comment by anas142** [comment]
I have same Problem with post request and @ahmad-shawky  solution  will be solved

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "laravel/telescope",
+    "name": "ahmad-shawky/telescope",
```
Comment (ManojKiranA) [inline composer.json]: Why Have You changed this

File: public/app-dark.css [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .vjs__tree{font-family:Monaco,Menlo,Consolas,Bitstream Vera Sans Mono,monospace!important}.vjs__tree .vjs__tree__content{border-left:1px dotted hsla(0,0%,80%,.28)!important}.vjs__tree .vjs__tree__node{cursor:pointer}.vjs__tree .vjs__tree__node:hover{color:#20a0ff}.vjs__tree .vjs-checkbox{position:absolute;left:-30px}.vjs__tree .vjs__value__boolean,.vjs__tree .vjs__value__null,.vjs__tree .vjs__value__number{color:#a291f5!important}.vjs__tree .vjs__value__string{color:#dacb4d!important}.hljs-addition,.hljs-keyword,.hljs-selector-tag{color:#8bd72f}.hljs-doctag,.hljs-meta .hljs-meta-string,.hljs-regexp,.hljs-string{color:#dacb4d}.hljs-literal,.hljs-number{color:#a291f5!important}
```
Comment (ManojKiranA) [inline public/app-dark.css]: kindly check this https://laravel.com/docs/5.8/contributions#compiled-assets

File: public/app-dark.css [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
```
Comment (ManojKiranA) [inline public/app-dark.css]: kindly check this https://laravel.com/docs/5.8/contributions#compiled-assets

File: composer.json (hunk 1 above)
Comment (ManojKiranA) [inline composer.json]: revert this

**Comment by taylorotwell** [comment]
I don't see the problem with using query string parameters.

**Pull request closed without merging**